components/collapse/nz-collapse-panel.component.html

```diff
@@ -1,5 +1,14 @@
 <div role="tab" [attr.aria-expanded]="nzActive" class="ant-collapse-header" (click)="clickHeader()">
```

Comment:
`<i>` tag would always be its first element child, you can use renderer here (an if users pass a template ref) to set the class.

Reply:
done